### ISSUE TITLE
Missing training set sheet in dataset.xslx?

### ISSUE THREAD

**Issue by JasonMoho** [sslab-gatech/SysGPT#1]
Great talk today at OSDI! 

I was curious about the contents of the training set. I downloaded the dataset.xslx file, but only saw a single sheet for the test set. 

It could be an issue on my end. I tried opening it in Numbers and on Google Sheets and in both cases only the test set came up.

**Comment by yicun0720** [comment]
The same issue on me.

**Comment by Panlichen** [comment]
The same issue on me, too.
Besides, can we get access to the code for fine-tuning the LLM with the training dataset?
Thanks a lot ^_^

**Comment by MaoZiming** [comment]
Same issue

**Comment by chenxuqiang** [comment]
Same issue
